src/client/QueueProblemType.ts

```diff
@@ -0,0 +1,4 @@
+export enum QueueProblemType {
```

Comment:
Similar thing as with the use of "API". When porting things from the Authority server, please make sure you use the correct terminology here.